src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message should indicate that it is a Bulldozer that is working. Consider specifying this in the message, for example: 'Bulldozer is working.'